python/openmldb/test/openmldb_client_test.py

```diff
@@ -395,6 +395,55 @@ def test_drop_table(self):
 
     def teardown_class(self):
         self.cursor.close()
+        
+class TestSQLMagicOpenMLDB:
+
+    def setup_class(self):
+        self.db = openmldb.dbapi.connect('db_test', '127.0.0.1:6181', '/onebox')
+        self.ip = openmldb.sql_magic.register(self.db)
+
+    def execute(self,magic_name,line):
+        try:
+            self.ip.run_line_magic(magic_name,line)
+            return 'ok'
+        except Exception as e:
+            raise Exception(e)
```

Comment:
If we just catch exception and raise, you can remove these lines and do not catch the exception.